src/Microsoft.DotNet.CodeFormatting/Rules/RuleOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.DotNet.CodeFormatter.Analyzers
+{
+    [Export(typeof(IOptionsProvider)), Shared]
+    internal class RuleOptions : IOptionsProvider
+    {
+        public IEnumerable<IOption> GetOptions()
+        {
+            return new List<IOption>
+            {
+                BraceNewLineEnabled,
+                CopyrightHeaderEnabled,
+                ExplicitThisEnabled,
+                ExplicitVisibilityEnabled,
+                FormatDocumentEnabled,
+                RemoveCustomCopyrightEnabled,
+                RemoveIllegalHeadersEnabled,
+                MarkReadonlyFieldsEnabled,
+                NewLineAboveEnabled,
+                UnicodeLiteralsEnabled,
+                UsingLocationEnabled
```

Comment:
Even though these options are not currently used, we probably still should add `ExplicitVariableTypeEnabled` here when resolving conflicts.